airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/conftest.py

```diff
@@ -126,6 +126,23 @@ def docker_runner_fixture(image_tag, tmp_path) -> ConnectorRunner:
     return ConnectorRunner(image_tag, volume=tmp_path)
 
 
+@pytest.fixture(name="previous_connector_image_name")
```

Comment:
Why not default to latest if the user doesn’t input it?

Also could you update the input docs in https://docs.airbyte.com/connector-development/testing-connectors/source-acceptance-tests-reference if this updates the input format?

Reply:
It's still defaulting to latest as `latest` is the default value for `previous_connector_version` input: 
in config.py: 
```python
previous_connector_version: str = Field(default="latest", description="Previous connector version to use for backward compatibility tests.")
```

Reply:
I just updated the reference doc.

Reply:
ah got it, didn't see that piece. not enough caffeine yet :) thanks!